doc/conf.py

```diff
@@ -92,9 +92,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.3'
+version = '0.4'
 # The full version, including alpha/beta/rc tags.
-release = '0.3.0'
+release = '0.4.0'
```

Comment:
I like to change the doc version when we do the pip release and upgrade the version in setup.py as well.